content/updates/2020-06/2-option-screen-revamp.md

```diff
@@ -0,0 +1,15 @@
+# New: option screen revamp
+::: meta-data type="Improvement" features="Data Governance" available="end-June" link-to-doc="../articles/manage-simple-and-multi-selects-attribute-options.html"
+:::
+
+Because single and multi-select attributes are in the top 5, we improved the 'option screen' user experience to ease creating or editing attribute options.
+
+Apart from a great design and improved visibility on content, nothing has changed: you can still create and edit options for all your activated locales.
+
+![Duplicate action in the `...` menu](../img/new-option-screen.png)
+
+We felt we could do more to help you ensure data quality and ease your way to a great PX Score... so we are planning new improvements for next month. Stay tuned :wink:
+
+::: more
+[Manage simple and multi selects attribute options](../articles/manage-simple-and-multi-selects-attribute-options)  
```

Comment:
Should be a question -> see here. https://www.notion.so/akeneo/Creating-Serenity-updates-content-86430cfb26ae4ee791052308f1586d67

Reply:
Link won't work, it's missing the ".html" part.